l3build-file-functions.lua

```diff
@@ -368,10 +367,10 @@ end
 
 -- Rename
 function ren(dir, source, dest)
-  local dir = dir .. "/"
+  dir = dir .. "/"
   if os_type == "windows" then
-    local source = gsub(source, "^%.+/", "")
-    local dest = gsub(dest, "^%.+/", "")
+    source = gsub(source, "^%.+/", "")
+    dest = gsub(dest, "^%.+/", "")
```

Comment:
both are already local as function arguments
BTW, there is a problem with the '+' in the pattern.
Either it is required, and we are allowed to refer to the parent directory, or useless.

Reply:
From memory, this was needed as Windows needs just names here, whereas Unix allows paths.

Reply:
What puzzles me is "^%.+/" instead of "^%./". If `source` starts with "../" this must be an error, no ?

Reply:
Ah, true